spring-cloud-skipper-client/src/test/java/org/springframework/cloud/skipper/client/DefaultSkipperClientTests.java

```diff
@@ -210,4 +213,38 @@ public void testReleaseList() {
 		List<Release> list = skipperClient.list(null);
 		assertThat(list).isEmpty();
 	}
+
+	@Test
+	public void testActuatorGet() {
+		RestTemplate restTemplate = new RestTemplate();
+		SkipperClient skipperClient = new DefaultSkipperClient("", restTemplate);
+
+		MockRestServiceServer mockServer = MockRestServiceServer.bindTo(restTemplate).build();
+		mockServer
+				.expect(requestTo("/release/actuator/tiktok/log/tiktok-log-0?endpoint=/foo/bar"))
+				.andExpect(queryParam("endpoint","/foo/bar"))
+				.andRespond(withSuccess("{\"foo\":\"bar\"}", MediaType.APPLICATION_JSON));
+
+
+		String response = skipperClient.getFromActuator(
+				"tiktok","log", "tiktok-log-0","/foo/bar");
+		assertThat(response).isEqualTo("{\"foo\":\"bar\"}");
+	}
+
+	@Test
+	public void testActuatorPost() {
+		RestTemplate restTemplate = new RestTemplate();
+		SkipperClient skipperClient = new DefaultSkipperClient("", restTemplate);
+		ActuatorPostRequest actuatorPostRequest = ActuatorPostRequest.of("/bindings/input",
+				Collections.singletonMap("state","STOPPED"));
+		MockRestServiceServer mockServer = MockRestServiceServer.bindTo(restTemplate).build();
+		mockServer
+				.expect(requestTo("/release/actuator/tiktok/log/tiktok-log-0"))
+				.andExpect(content().json(
+						"{\"endpoint\":\"/bindings/input\",\"body\":{\"state\":\"STOPPED\"}}"))
+				.andRespond(withSuccess());
+
+		Object response = skipperClient.postToActuator(
+				"tiktok","log", "tiktok-log-0", actuatorPostRequest);
```

Comment:
I think you forgot the asserts ;)

Reply:
the `expects` are actually asserts on the request. I don't care about the response.

Reply:
Ahh yeh, got it. Do you mind adding this as a 1 liner comment?